ghost/core/core/frontend/services/theme-engine/middleware/update-global-template-options.js

```diff
@@ -16,6 +15,13 @@ function getSiteData() {
         siteData.signup_url = '#/portal';
     }
 
+    // theme only property added to @site
```

Comment:
To me this is a discrete change again. I'd make it in a separate PR. Probably first as it's the simplest.